README.md

```diff
@@ -29,11 +29,33 @@ await register({
   options: {
     prefix: '/v1',
     name: 'db', // the same name you used for configuring `hapi-sequelize` (options.name)
-    defaultConfig: { ... } // passed as `config` to all routes created
+    defaultConfig: { ... }, // passed as `config` to all routes created
+    models: ['cat', 'dog'] // only the cat and dog models will have routes created
```

Comment:
Please note that omitting the `models` property enables routes for all models, something like this:

```
// You can specify which models must have routes defined for using the `models` property.
// If you omit this property, all models will have models defined for them.
```